app/src/androidTest/java/org/mozilla/focus/activity/screenshots/ScreenGrabTest.java

```diff
@@ -177,7 +175,7 @@ public void screenGrabTest() throws InterruptedException, UiObjectNotFoundExcept
         takeScreenshotOfErrorPages(context, device);
     }
 
-    private void takeScreenshotsOfFirstrun(Context context, UiDevice device) throws UiObjectNotFoundException {
+    private void takeScreenshotsOfFirstrun(Context context, UiDevice device) throws UiObjectNotFoundException, InterruptedException {
```

Comment:
I opened a new PR for separating this test class into multiple classes. Let's remove those changes here and apply them to the new classes if needed.

Reply:
agree, will wait until https://github.com/mozilla-mobile/focus-android/pull/1505 and https://github.com/mozilla-mobile/focus-android/pull/1480 are merged.